AgileFront/src/app/services/post.ts

```diff
@@ -3,4 +3,5 @@ export interface Post {
   body?: string;
   imageURL: string;
   userId: string;
+  postId?: string;
```

Comment:
I get why you do that, might be a more elegent way to do so, but whatev, one week left anyway

Reply:
Yeah I said it in the comment on the PR